apps/omg_child_chain/test/omg_child_chain/block_queue/core_test.exs

```diff
@@ -186,6 +186,7 @@ defmodule OMG.ChildChain.BlockQueue.CoreTest do
                Core.new(
                  mined_child_block_num: 2000,
                  known_hashes: [{1000, <<1::size(256)>>}],
+                 # TODO(pdobacz, fixing in a follow-up PR): shouldn't be 1 below?
```

Comment:
No, I think it is consistent with `mined_child_block_num: 2000`. This scenario can be interpreted as "block was mined and submitted, but write to DB has failed" or alternatively "someone else has submitted the block for us".

Reply:
ok gotcha, you're right. I'll then keep both this version and one with `1` in the hash. I think both tests are relevant. And I'll ditch the duplicate https://github.com/omisego/elixir-omg/pull/1253/files#diff-517a845b00e424eeec3da45ca99c9450R218